test/vast_parser.js

```diff
@@ -292,6 +292,35 @@ describe('VASTParser', function() {
           linear.duration.should.equal(90.123);
         });
 
+        it('should have a universal ad id', () => {
```

Comment:
we are in the process of migrating unit tests from mocha to jest. Since you are adding new functionality I would prefer it if you make the tests in jest. Same for future PRs ;)

Reply:
Have added this to the Creatives parser spec and the Extensions parser spec as well.